packages/wallet/backend/src/transaction/validation.ts

```diff
@@ -5,3 +5,22 @@ export const transactionListRequestSchema = z.object({
     orderByDate: z.enum(['ASC', 'DESC']).default('DESC')
   })
 })
+
+const transactionExtendedObject = z
+  .object({
+    accountId: z.string().uuid().optional(),
+    paymentPointerId: z.string().uuid().optional(),
+    assetCode: z.string().optional(),
+    type: z.string().optional(),
+    status: z.string().optional()
+  })
+  .strict()
+
+export const transactionListAllRequestSchema = z.object({
+  query: z.object({
+    page: z.coerce.number().int().nonnegative().default(0),
```

Comment:
default 1?
on first request page 1 will be returned so it make sense for default to be 1

Reply:
https://vincit.github.io/objection.js/api/query-builder/other-methods.html#page